tests/deployment/test_deployment_utils.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from subprocess import CalledProcessError
+import subprocess
+
+from bentoml.deployment.utils import ensure_docker_available_or_raise
+from bentoml.exceptions import BentoMLMissingDependencyException, BentoMLException
+
+
+def raise_(ex):
+    raise ex
+
+
+def test_ensure_docker_available_or_raise():
```

Comment:
use mock instead of setattr

Reply:
I tried a lot of ways with mock, no luck.